storage/mongo/invite_storage.go

```diff
@@ -100,7 +100,7 @@ func (is *InviteStorage) GetAll(withArchived bool, skip, limit int) ([]model.Inv
 	defer cancel()
 
 	filter := bson.M{}
-	if withArchived == false {
+	if !withArchived {
```

Comment:
Explicit declaration like `== false` is more preferable in go

Reply:
Linter did it